model/manifest/manifest.go

```diff
@@ -9,13 +9,19 @@ const Collection = "manifest"
 // Branch is the branch of the repository. Modules is a map of the GitHub repository name to the
 // Module's information associated with the specific version.
 type Manifest struct {
-	Id              string             `json:"id" bson:"_id"`
-	Revision        string             `json:"revision" bson:"revision"`
-	ProjectName     string             `json:"project" bson:"project"`
-	Branch          string             `json:"branch" bson:"branch"`
-	Modules         map[string]*Module `json:"modules" bson:"modules"`
-	IsBase          bool               `json:"is_base" bson:"is_base"`
-	ModuleOverrides map[string]string  `json:"module_overrides,omitempty" bson:"-"`
+	// Identifier for the version.
+	Id string `json:"id" bson:"_id"`
+	// The revision of the version.
+	Revision string `json:"revision" bson:"revision"`
+	// The project identifier for the version.
+	ProjectName string `json:"project" bson:"project"`
+	// The branch of the repository.
+	Branch string `json:"branch" bson:"branch"`
+	// Map from the Github repository name to the module's information.
+	Modules map[string]*Module `json:"modules" bson:"modules"`
+	// 	True if the version is a mainline build.
```

Comment:
style nit: one space between `//` and the start of the comment.

Same with the rest of the fields that have two spaces.